packages/readiness/bundle/config/upstream/rbac.yaml

```diff
@@ -0,0 +1,60 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: tanzu-readiness-manager
+  annotations:
+    kapp.k14s.io/change-group: "readiness.core.tanzu.vmware.com/serviceaccount"
+  name: tanzu-readiness-manager-sa
+  namespace: #@ data.values.namespace
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    kapp.k14s.io/change-group: "readiness.core.tanzu.vmware.com/serviceaccount"
+  name: tanzu-readiness-manager-clusterrole
+rules:
```

Comment:
We need to extend the rules so that it could read any resource of any type.

Reply:
We can do that later though!

Reply:
Yes, right now they only contain access rules specific to readiness resource. We can keep updating this list as we go.

Reply:
What resources are we expecting to read here? Anything in the cluster?

Reply:
If it's "any" resource in the cluster, you should consider taking a service account as an input in the API instead of adding a `*` rule here, which is not best practice. [See how Capability API does this](https://github.com/vmware-tanzu/tanzu-framework/blob/6a32fdc564d967d9cd64d96a1af11aa55d54f860/apis/core/v1alpha2/capability_types.go#L13-L19).

This makes the user responsible for providing the right RBAC.

Reply:
Thanks @rajathagasthya .. this makes sense!